src/ui/toolbar/Toolbar.vue

```diff
@@ -138,7 +143,7 @@ export default {
         domainObject,
         '*',
         function (newObject) {
-          this.domainObjectsById[id].newObject = JSON.parse(JSON.stringify(newObject));
```

Comment:
I wonder why we were doing this?

Reply:
🤷‍♂️

we shouldn't have been

Reply:
🤷‍♂️

we shouldn't have been

Reply:
yeah, I've ripped out hacky stuff before and found out later why it needed to be there 🥲

If there was some reason, this will force us to find it!